src/tex/ms.tex

```diff
@@ -166,8 +169,9 @@ \section{Prior}
 \end{figure}
 %
 
-In the case that the prior distribution cannot be described easily with some mathematical function, for example, if it is a multimodal model result from some other sampling method, then the chain from the prior sampling should be given. 
-The chain is all of the samples investigated in the sampling and should be shared, although in the case that this chain is very large a subsampled object may be shared, in which case, the autocorrelation analysis performed should be described (see Section~\ref{sec:posterior} for a more complete discussion of this).
+It is possible that the prior distribution cannot be described with a simple mathematical function, with multi-modal priors being an example.
```

Comment:
Not all multi-model priors cannot be described with a mathematical function though.